pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -120,6 +124,19 @@ def __init__(self) -> None:
         )
 
 
+class GateSet(Enum):
+    X_SX_RZ_CX = {OpType.X, OpType.SX, OpType.Rz, OpType.CX}
+    X_SX_RZ_ECR = {OpType.X, OpType.SX, OpType.Rz, OpType.ECR}
+    UNKNOWN = None
```

Comment:
Maybe the None is redundant here now? In the `UnsupportedGateSetException` I want to give the user the unsupported OpTypes as I think this is more helpful.